cmd/credential/password/password.go

```diff
@@ -49,21 +52,27 @@ const (
 	FlagNamePasswordCount            = "count"
 )
 
+// [Q]: don't get why did you use this interface.
```

Comment:
I had a plan in mind but later on dropped the idea but forgot to remove the code defining the interface. For now, i have just removed the interface as well as the struct also as they are not serving any purpose.